README.md

```diff
@@ -1,2 +1,33 @@
 # Exam-Attendance-UI
-This the front end app for the exam attendance
+
+This is the front end app for the exam attendance.
+
+# Project Screen Shot(s)
+
+![Alt text](./src/images/image-1.png)
+![Alt text](./src/images/image-2.png)
+Once examiner logs in he/she can view the exams on current day.
+![Alt text](./src/images/image-3.png)
+Once the examiner press on an exam. Examiner will be directed to the attendance monitoring page.
+![Alt text](./src/images/image-4.png)
+Once examiner press `Start Attending` on `Attendance Monitoring` page. This window will be opened to take attendance of students.
+![Alt text](./src/images/image-5.png)
+In case, student can be verified by the index as well.
```

Comment:
Isn't that better to put some sub headings above the screen shots and small description about that page.

Reply:
Sure. I'll add.